src/style.css

```diff
@@ -1 +1,21 @@
-/* styles go here */
+.label {
+  display: block;
+  margin-bottom: 10px;
+}
+
+.age input {
```

Comment:
do not style tags
<img width="422" alt="image" src="https://user-images.githubusercontent.com/102411130/228765310-9c43a9c8-6f44-4bf9-adf0-cf20c19dad16.png">